src/components/common/BigNumberInput/index.tsx

```diff
@@ -0,0 +1,79 @@
+import { BigNumber, formatUnits, parseUnits } from 'ethers/utils'
```

Comment:
If we are not fixing the error in this file I think is best to use BigNumberInput from dependencies instead of having this file

Reply:
@lmcorbalan there was an small change in the line 31 of the file, I didn't know if this happened because react hook form or what, but sometimes the value is '0' (a string)

Reply:
I added again the library, and an small check in the BigNumberInputWrapper so the value is no more a zero string